kod/object/item/passitem/spelitem/potion.kod

```diff
@@ -135,13 +142,13 @@ messages:
                  #duration=(POISON_DURATION*Random(8,12)/10));
          }
 
-         send(self,@Delete);
```

Comment:
Potion no longer is deleted after using it?

Reply:
It gets deleted when its piHits value reaches 0, which is handled in the parent class's OnSuccessfulApply